06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/lib/css/custom.css

```diff
@@ -0,0 +1,347 @@
+ .image-container {
+    min-height: 100vh;
+    background-position: center center;
+    background-size: cover;
+    position: relative;
+    }
+
+    body {
```

Comment:
inconsistent indentation and position of curly braces in this file, and move body {} to the top